appveyor.yml

```diff
@@ -0,0 +1,40 @@
+environment:
+  password:
+    secure: uOcv5TOm/CgwOmiLQhNX+g==
+
+  matrix:
+    - python : 27
+    - python : 27-x64
+    - python : 33
+    - python : 33-x64    
+    - python : 34
+    - python : 34-x64
+    - python : 35	
+    - python : 35-x64
+
+install:
+  - "SET PATH=C:\\Python%PYTHON%;c:\\Python%PYTHON%\\scripts;%PATH%"
+  - echo "C:\Program Files\Microsoft SDKs\Windows\v7.1\Bin\SetEnv.cmd" /x64 > "C:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\bin\amd64/vcvars64.bat"
```

Comment:
That slash in `/vcvarsall.bar` at the end looks strange after all the other paths used backslashes.